### ISSUE TITLE
passphrase not recognised

### ISSUE THREAD

**Issue by PaulS59** [Sneezry/authenticator#119]
Having had this app on Google Chrome I am all of a sudden getting the message that my passphrase is not recognised for some accounts.  How can I work around this as I need access to these accounts

**Comment by Brianetta** [comment]
Restore your backup. I have to do this every couple of weeks.

**Comment by kardinal23** [comment]
This is very timeconsuming bug. At first I have to say that for years a password was useless since regardless what password I chose, it could be easily overdone by clicking on set a password and I made a new one....  without password its much easier.

whats the use of having a password on a notebook which is encrypted by truecrypt and no one else except of me has acces to it?? why I should have a password to this..... How can I disable this buggy feature. I dont want to have password...

**Comment by kardinal23** [comment]
How can I restore backup mr oracle Brianetta?
I dont see it in menu

**Comment by Brianetta** [comment]
If you set a password, the extension can synchronise through Google services.

To backup, click the cog, then Import/Export. You can copy the entire database, and save it in a secure location. You can restore by pasting into this text box, overwriting the contents.

**Comment by kardinal23** [comment]
Thank you but it doesnot work
I can see export import
I can copy it or delete.... but I copied it, delete, then paste it in a window again and click on update and it is still the same, I dont see codes but asterics and when I move mouse over asterics I cas see red notice You dont have a password set...
Ok I click on set a password and notjing happens if I put that correct password...

**Comment by Brianetta** [comment]
Unfortunately, you've hit this bug. I always make my password blank before copying out the codes for backup, and then put my password back again afterwards. If you don't have a working backup, you'll just have to add each account again from scratch.

**Comment by kardinal23** [comment]
I would set up my password as a blank nothing however when I try to do it, situation is still the same = I see asterics only..
I found another 4 addons
https://chrome.google.com/webstore/search/google%202FA?
SAASPASS and others
which one could be better?

**Comment by kardinal23** [comment]
OTP safe is simple unfortunately the other are complicated
But I cannot export coc*****ing secret keys
what I see is probably encrypted or changed.... 
......
At first - I didnot set any password for original authenticator
then application asked for a password so I set something and now it refuses to work.... I will delete such useless addon but at first I must export data away and I dont know how

**Comment by kardinal23** [comment]
That other notebook asked for a password as well... so I have it unaccessible... WTF????? will it fix somehow? that addon worked well for 3 years and now this shit happen? 
well this bug make the whole addon quite useless and also dangerous since I cannot get to accounts. You cant imagine how upset am I 
in a mobile I am afraid I have the same shit.... if I loose it too I loose acceess to my accounts. WTF?

**Comment by kardinal23** [comment]
how do I get to that secrets from accounts? I must get them before something happens to the application on my mobile
but mobile app call authenticator doesnot have export option in menu.....

**Comment by Sneezry** [comment]
@kardinal23 

Before you install this extension, you can see this in web store

![snipaste20170524_100951](https://cloud.githubusercontent.com/assets/1621293/26384101/8f0ccc8c-4069-11e7-9923-57828f459cc6.png)

After you installed the extension, you can see this every 30 days
![snipaste20170524_101155](https://cloud.githubusercontent.com/assets/1621293/26384122/a126079e-4069-11e7-81fa-34d1a8572d97.png)

What can I do more? Call your phone to ask you to backup your data every week?

**Comment by SinanWang** [comment]
hahahahahaha

**Comment by Brianetta** [comment]
Good news: Kardinal23 now has access to his secrets, after finding and backing up an unsynchronized instance that had no password. He got lucky.

**Comment by daviestang** [comment]
From the issue description, is there some bugs in code caused the data corruption. I think even user did not backup their data, the program also shouldn't break the user's accounts data.

**Comment by mazyi** [comment]
v2ex sightseeing party

**Comment by Brianetta** [comment]
Yes. I managed to repair mine. I went into the export/import window and removed the most recently added account from the json list. It then decrypted the rest without issue.
